tests/Doctrine/Tests/DBAL/Platforms/MySQL57PlatformTest.php

```diff
@@ -78,7 +78,7 @@ protected function getQuotedAlterTableRenameIndexInSchemaSQL()
     protected function getGeneratesAlterTableRenameIndexUsedByForeignKeySQL()
     {
         return array(
-            'ALTER TABLE mytable RENAME INDEX idx_foo TO idx_foo_renamed',
+            'ALTER TABLE mytable RENAME INDEX `idx-foo` TO `idx-foo-renamed`',// Index name with special character in name (needs quotation on some platforms, e.g. Sqlite).
```

Comment:
Have you seen http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/basic-mapping.html#quoting-reserved-words and http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/reference/limitations-and-known-issues.html#identifier-quoting-and-legacy-databases ?